appinventor/components-ios/src/Sprite.swift

```diff
@@ -311,25 +379,25 @@ open class Sprite: ViewComponent, UIGestureRecognizerDelegate {
     if (normalizedAngle < 0) {
       normalizedAngle += 360
     }
-
+    
     // Only transform heading if sprite was moving in that direction.
     // This avoids oscillations.
     if ((e == Direction.east
-      && (normalizedAngle < 90 || normalizedAngle > 270))
-      || (e == Direction.west
-        && (normalizedAngle > 90 && normalizedAngle < 270))) {
+         && (normalizedAngle < 90 || normalizedAngle > 270))
+        || (e == Direction.west
+            && (normalizedAngle > 90 && normalizedAngle < 270))) {
       Heading = 180 - normalizedAngle
     } else if ((e == Direction.north
-      && normalizedAngle > 0 && normalizedAngle < 180)
-      || (e == Direction.south && normalizedAngle > 180)) {
+                && normalizedAngle > 0 && normalizedAngle < 180)
+               || (e == Direction.south && normalizedAngle > 180)) {
       Heading = 360 - normalizedAngle
     } else if ((e == Direction.northeast
-      && normalizedAngle > 0 && normalizedAngle < 90)
-      || (e == Direction.northwest
-        && normalizedAngle > 90 && normalizedAngle < 180)
-      || (e == Direction.southwest
-        && normalizedAngle > 180 && normalizedAngle < 270)
-      || (e == Direction.southeast && normalizedAngle > 270)) {
+                && normalizedAngle > 0 && normalizedAngle < 90)
+               || (e == Direction.northwest
+                   && normalizedAngle > 90 && normalizedAngle < 180)
+               || (e == Direction.southwest
+                   && normalizedAngle > 180 && normalizedAngle < 270)
+               || (e == Direction.southeast && normalizedAngle > 270)) {
```

Comment:
All these additional spaces seems unnecessary, can you restore them?

Reply:
I removed the extra spaces. They were probably there from before.